app/experimenter/nimbus-ui/src/components/SidebarActions/index.tsx

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { RouteComponentProps } from "@reach/router";
+import React from "react";
+import { getExperiment_experimentBySlug } from "../../types/getExperiment";
+import { LinkNav } from "../LinkNav";
+import "./index.scss";
+import { ReactComponent as Trash } from "./trash.svg";
+
+type SidebarModifyExperimentProps = {
+  testid?: string;
+  experiment: getExperiment_experimentBySlug;
+} & RouteComponentProps;
+
+export const SidebarActions = ({
+  experiment,
+  testid = "SidebarActions",
+}: SidebarModifyExperimentProps) => {
+  return (
+    <div data-testid={testid}>
+      <p className="edit-divider position-relative small my-2">
+        <span className="position-relative bg-light pl-1 pr-2 text-muted">
+          Actions
```

Comment:
Oh forgot one more. I love that we're re-using this sections pattern! Since the title in the section above it contains the word "Experiment" can we make this consistent by either calling this "Experiment Actions", or calling the above section simply "Edit"?

![Screen Shot 2021-08-11 at 3 40 40 PM](https://user-images.githubusercontent.com/6392049/129086278-06f8c858-4a56-4326-a71c-410119e19f03.png)

Reply:
Yeah I was trying to think of how to keep those consistent but 

```
Edit Experiment
Experiment Actions
```

literally killed me to look at.  But yes I like dropping `Experiment` from edit!